README.md

```diff
@@ -174,28 +174,28 @@ useminPrepare: {
 
 ### dest
 
-Type: 'string'  
```

Comment:
Sorry this was sublime text doing its thing. I remember reverting these changes before committing, not sure how they got through.